### PR TITLE
buildah source pull/push: show progress bar

### PR DESCRIPTION
**Pull request by vrothberg**
Render the progress bar on stdout by default.  Add a --quiet/-q option
to re-silence if desired.

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

@rhatdan PTAL

### CODE REVIEW

File: docs/buildah-source-pull.1.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--quiet**, **-q**
+
+Suppres the progress output when pulling a source image.
```
Comment (edsantiago) [inline docs/buildah-source-pull.1.md:26]: Typo: `suppress` with two final esses. Same below.

**Review by edsantiago — COMMENTED** [review]
What is the context for this? I'm ultra-uncomfortable with using stdout; would much rather see it on stderr like podman. I'm surprised tests aren't failing.

**Comment by vrothberg** [comment]
> What is the context for this? I'm ultra-uncomfortable with using stdout; would much rather see it on stderr like podman. I'm surprised tests aren't failing.

Thanks for checking.  My heat-fried brain let me believe it bars on stdout but you are absolutely right that they belong in stderr; will update.

**Pull request merged**